dandischema/exceptions.py

```diff
@@ -7,20 +8,14 @@ class ValidationError(ValueError):
 
 
 class JsonschemaValidationError(ValidationError):
-    """Validation errors were detected by jsonschema.
+    """Validation errors were detected by jsonschema"""
 
-    All errors are contained in the .args[0] of the exception instance.
-    """
-    @property
-    def errors(self) -> List[jsonschema.exceptions.ValidationError]:
-        return self.args[0]
+    def __init__(self, errors: List[jsonschema.exceptions.ValidationError]) -> None:
+        self.errors = errors
```

Comment:
doesn't this change the semantics from something that is readonly to something that is editable?

Reply:
Yes.  Is that a problem?